packages/cli/src/resource/strings.json

```diff
@@ -28,6 +28,23 @@
         "force": "Force upgrade the project to work with the latest version of Teams Toolkit."
       },
       "success": "Upgrade project successfully."
+    },
+    "doctor": {
+      "account": {
+        "SideLoadingDisabled": "Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams!",
+        "NotSignIn": "No Microsoft 365 account login",
+        "SignInSuccess": "Microsoft 365 Account (%s) is logged in and sideloading permission enabled."
```

Comment:
NoSignIn: `You have not logged in to your Microsoft 365 account yet. Please use teamsapp auth login command to login to your Microsoft 365 account.`